upstairs/src/downstairs.rs

```diff
@@ -3011,15 +3016,15 @@ impl Downstairs {
                 }
                 IOop::ExtentFlushClose { extent, .. } => {
                     let job_type = "FClose".to_string();
-                    (job_type, *extent)
+                    (job_type, extent.0 as usize) // XXX this isn't blocks!
```

Comment:
Hmm, make it zero instead?

Reply:
Yup, done.

We noted in chat that `ioop_summary` is also returning the ambiguous blocks / extent field, but it's printed as "BL/E" so I left it alone for now.

Reply:
Yeah, that was me being lazy and re-using a column instead of making a new one.
All the repair operations overload that blocks column and use it for reporting extent id.

Reply:
Rather than just print 0 in this column, why not change the column to be 'BL/E' like `ioop_summary` so we
can still print and see the information here?

Reply:
Okay, done in 43c97ee42704 (using `BKS/EXT` as the column header, since we have 7 characters to play with)